code/script.js

```diff
@@ -1,9 +1,12 @@
+
 // A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
```

Comment:
Maybe this`// console.log(this) to see how it works` comment doesn't need to be here?